poetry.lock

```diff
@@ -3276,69 +3385,49 @@ files = [
 ]
 
 [[package]]
-name = "numpy"
-version = "2.0.2"
-description = "Fundamental package for array computing in Python"
+name = "numba"
+version = "0.61.2"
+description = "compiling Python code using LLVM"
 optional = true
-python-versions = ">=3.9"
```

Comment:
it looks like numpy for 3.9 is removed here

Reply:
Numpy is here: https://github.com/ehsantn/iceberg-python/blob/f36265b8cdc9fa3056ad28784467579514cfc850/poetry.lock#L3424
I'm working on packaging Bodo for Python 3.9 to avoid these Poetry issues: https://github.com/bodo-ai/Bodo/pull/637
Our team will just miss structured pattern matching and better type hints of Python 3.10 :)

Reply:
sg 3.9 is also EOL in a few months (2025-10) 
https://devguide.python.org/versions/#supported-versions